### PR TITLE
Responsive Style

### PR DESCRIPTION
**Pull request by hassanelnajjar**
# Responsive Style
relates #113

### CODE REVIEW

File: client/src/components/common/List/index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { List } from 'antd';
+
+const customList = ({ dataSource, ...otherProps }) => (
```
Comment (Mu7ammadAbed) [inline client/src/components/common/List/index.js:4]:
```suggestion
const CustomList = ({ dataSource, ...otherProps }) => (
```

File: client/src/components/common/List/index.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+          avatar={item.icon}
+          title={item.title}
+          description={item.description}
+        />
+      </List.Item>
+    )}
+  />
+);
+
+export default customList;
```
Comment (Mu7ammadAbed) [inline client/src/components/common/List/index.js:20]:
```suggestion
export default CustomList;
```

File: client/src/pages/home/Map/style.css [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.map-icon{
+    color:#52CDDE;
+		font-size: 2rem;
+}
+
```
Comment (Mu7ammadAbed) [inline client/src/pages/home/Map/style.css:5]: Fix these

File: client/src/pages/logIn/style.css [hunk 4]
```diff
@@ -1,14 +1,9 @@
...
-    width:600px;
-    height: 350px;
-    border-radius: 5px;
-}
+/* 
+
+.loginFormStyle{
+    width: 400px;
+} */
+
```
Comment (Mu7ammadAbed) [inline client/src/pages/logIn/style.css:6]: ??

Comment (hassanelnajjar) [reply client/src/pages/logIn/style.css:6]: Removed :)

File: client/src/pages/home/Map/style.css (hunk 3 above)
Comment (hassanelnajjar) [reply client/src/pages/home/Map/style.css:5]: Done

File: client/src/components/common/List/index.js (hunk 2 above)
Comment (hassanelnajjar) [reply client/src/components/common/List/index.js:20]: Done

File: client/src/components/common/List/index.js (hunk 1 above)
Comment (hassanelnajjar) [reply client/src/components/common/List/index.js:4]: Done

**Review by Mu7ammadAbed — APPROVED** [review]

**Pull request merged**